scripts/common/cmdline_helpers.py

```diff
@@ -19,7 +20,9 @@ def f(*args, **kwargs):
             try:
                 tmp_dir = mkdtemp(prefix=prefix)
                 os.chdir(tmp_dir)
-                return fn(*args, **kwargs)
+                result = fn(*args, **kwargs)
+                rmtree(tmp_dir)
```

Comment:
Oh! Nice catch, I forgot that `mkdtemp` does not automatically remove the directory.